velox/type/Conversions.h

```diff
@@ -166,19 +166,16 @@ struct Converter<
     T result = 0;
     int index = 0;
     int len = v.size();
-    if (len == 0) {
-      VELOX_USER_FAIL("Cannot cast an empty string to an integral value.");
-    }
+    VELOX_USER_CHECK_GT(
```

Comment:
I would suggest not to introduce irrelevant change.
And `VELOX_USER_CHECK_EQ` is same semantic change. `GT` is also OK for me.